translations/en.i18n.json

```diff
@@ -58,7 +58,8 @@
       "qualityHdLabel": "Quality is HD",
       "quality4kLabel": "Quality is 4k",
       "quality8kLabel": "Quality is 8k",
-      "advancedAudioDescription": "Advanced Audio Description"
+      "advanced_audio_description_enabled": "Disable Extended Audio Description",
```

Comment:
just a reminder about ticket for other langs

Reply:
added, thanks